plugins/github-pull-requests-board/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@backstage/plugin-github-pull-requests-board",
+  "description": "A Backstage plugin that allows you to see all open Pull Requests for all the repositories owned by your team",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/github-pull-requests-board"
+  },
+  "keywords": [
+    "backstage",
+    "github",
+    "pull requests"
+  ],
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^1.0.1",
+    "@backstage/core-components": "^0.9.3",
+    "@backstage/core-plugin-api": "^1.0.1",
+    "@backstage/plugin-catalog-react": "^1.0.1",
+    "@backstage/theme": "^0.2.15",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "@octokit/rest": "^18.6.7",
```

Comment:
I think we could set version ^18.12.0 here, since this will install that version anyways, and we could spare a yarn.lock addition.